src/cloud-api-adaptor/test/e2e/common_suite.go

```diff
@@ -595,13 +595,14 @@ func DoTestKbsKeyRelease(t *testing.T, e env.Environment, assert CloudAssert) {
 func DoTestKbsKeyReleaseForFailure(t *testing.T, e env.Environment, assert CloudAssert) {
 
 	log.Info("Do test kbs key release failure case")
-	pod := NewBusyboxPodWithName(E2eNamespace, "busybox-wget-failure")
+	pod := NewCurlPodWithName(E2eNamespace, "curl-failure")
 	testCommands := []TestCommand{
 		{
-			Command:       []string{"wget", "-q", "-O-", "http://127.0.0.1:8006/cdh/resource/reponame/workload_key/key.bin"},
+			Command:       []string{"curl", "-s", "http://127.0.0.1:8006/cdh/resource/reponame/workload_key/key.bin"},
 			ContainerName: pod.Spec.Containers[0].Name,
 			TestCommandStdoutFn: func(stdout bytes.Buffer) bool {
-				if strings.Contains(stdout.String(), "request unautorized") {
+				body := stdout.String()
+				if strings.Contains(strings.ToLower(body), "error") {
```

Comment:
I think it would be good to have a more specific error, but I think that depends on https://github.com/confidential-containers/guest-components/issues/587, so this is okay for now.